05/DB.test.js

```diff
@@ -1 +1,92 @@
-import DB from './DB';
\ No newline at end of file
+import DB from './DB';
+
+describe(`DB`, () => {
+    const db = new DB();
+    describe('insert()', () => {
+        it(`if function invoked without argument`, () => {
+            expect(() => { db.insert() }).toThrow();
+        });
+
+        it(`if data.id is not a number`, async () => {
+            await expect(db.insert({ 'id': 'a' })).rejects.toBe('ID can be only number!');
+        });
+
+        it(`if id will duplicated`, async () => {
+            const fakeData = { 'a': 'abc', 'id': 2 };
+            db._rows.push(fakeData);
+
+            await expect(db.insert({ 'b': 'def', 'id': 2 })).rejects.toBe('ID can\'t be duplicated!');
+        });
+
+        it(`id doesn't exist`, async () => {
+            await expect(db.insert({ 'a': 'abc' })).resolves.toStrictEqual({ 'a': 'abc', 'id': 3 })
+        });
+    });
+    describe('select()', () => {
+        it(`id is not a number`, () => {
+            expect(() => { db.select('a') }).toThrow();
+        });
+
+        it(`select method doesn't has argument`, () => {
+            expect(() => { db.select() }).toThrow();
+        });
+
+        it(`id not find`, async () => {
+            db._rows = [{ 'a': 'abc', 'id': 2 }];
```

Comment:
Ponieważ, mamy podkreślenie przy nazwie właściwości to nie powinniśmy się odnosić do tej właściwości z poza wnętrza tego obiektu (klasy). Taka konwencja, że ta właściwość jest prywatna (dostępna tylko dla tego obiektu z wewnątrz.
Jeśli chcesz dodać elementy użyj `insert()`